app.vup.Vup.yaml

```
@@ -0,0 +1,78 @@
+app-id: app.vup.Vup
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory : "lib/ffmpeg"
+    version: "21.08"
```

```suggestion
    version: "22.08"
```

Gnome 43 use 22.08

But do you really need the GNOME runtime?

---

I do not, I added it to get libsecret out of the box but it seems I can easily get that as a submodule so I'll switch it to freedesktop.